### ISSUE TITLE
June Dependency Updates

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#6381]
We should try to review and merge in dependency updates on a monthly basis. Until this is automated more, here is a guide:

- Review the [outstanding dependency updates](https://github.com/uiowa/uiowa/labels/dependencies) and for ones that we plan to address in this epic, list them below. Some are PRs by @dependabot, some are issues.
- Dependencies related to our frontend build tools (not executed on the websites) are usually safe to merge as-is but might cause issue with other packages. Check travis build for messages related to `blt frontend`.
- Each @dependabot PR can be worked on separately. Each dependency PR you merge will cause merge conflicts with all the other ones. As long as you don't alter the branch (e.g. merge in `main`, push a commit), you can fix the merge conflicts by running `@dependabot rebase`.
- If a @dependabot PR does require some type of modification (e.g. removing a patch, updating incompatible code), it is okay to make those changes directly in relevant PR. However, it is a good idea to wait until you have merged all the other "easy" dependency PRs first because you will lose the ability to use `@dependabot rebase` to have the PR's merge conflicts resolved.
- Periodically check [locked dependencies](https://github.com/uiowa/uiowa/blob/main/README.md#locked-packages) for new versions. Make notes about why a dependency was/is locked in the repo Readme.
- Review the changelog for each version we are jumping to and the versions in-between. Make note of certain functionality in our codebase that is using the functionality for testing instructions later.
- It can also be helpful to review the git diff comparison between the old version and the new one to see how extensive the code changes are and to get an idea of what things you might need to test.
- Use https://github.com/uiowa/uiowa#updating-dependencies as a guide to test for update hooks, configuration changes and commit. This command is helpful if there are a lot of configuration changes across feature/site splits: https://github.com/uiowa/uiowa/blob/main/blt/src/Blt/Plugin/Commands/ConfigSplitCommands.php
- Dependency PR's with update hooks or requiring a cache rebuild may require off-hours deployment. If this is the case, add the appropriate label to the PR.
- Once this issue is completed, copy these notes to the next month's issue and introduce it in parking lot.

**Comment by joewhitsitt** [comment]
As noted earlier, it would be good to focus on dependencies that move us towards https://github.com/uiowa/uiowa/issues/6061

**Comment by joewhitsitt** [comment]
Focusing first on frontend build dependencies and locked packages (https://github.com/uiowa/uiowa/blob/main/README.md#locked-packages)

by way of

https://github.com/uiowa/uiowa/issues/5664
https://www.drupal.org/project/media_thumbnails_video/releases/2.0.0
https://www.drupal.org/project/purge/releases/8.x-3.4

**Comment by pyrello** [comment]
I'm going to work on updating `drupal/linkit` which has not been picked up by dependabot, but has available updates. This will help us when we get to #6537.

**Comment by pyrello** [comment]
@joewhitsitt Do we want to consider #6542 as part of this? If not, I think this is done once #6533 is merged.

**Comment by pyrello** [comment]
We discussed in EoD having #6542 be done as its own issue because there is more discovery to be done and we may want to rethink how we approach it since it may involve updating our AC platform email implementation. So, I think this is done. Nice work everyone!
